stdlib/selectors.pyi

```diff
@@ -31,49 +31,41 @@ class BaseSelector(metaclass=ABCMeta):
     def __enter__(self) -> Self: ...
     def __exit__(self, *args: Unused) -> None: ...
 
-class SelectSelector(BaseSelector):
+class _BaseSelectorImpl(BaseSelector, metaclass=ABCMeta):
+    def __init__(self) -> None: ...
```

Comment:
Nit: do we need this? In typeshed we usually skip methods that match the super-class's.

Reply:
good catch; I can drop close() from this class too